frontend/src/components/GddSend/TransactionForm.vue

```diff
@@ -179,13 +187,15 @@ export default {
         amount: Number(this.form.amount.replace(',', '.')),
         memo: this.form.memo,
         userName: this.userName,
+        targetCommunity: this.form.targetCommunity,
       })
     },
     onReset(event) {
       event.preventDefault()
       this.form.identifier = ''
       this.form.amount = ''
       this.form.memo = ''
+      this.form.targetCommunity = { id: 0, name: COMMUNITY_NAME }
```

Comment:
is this correct to set the default selected community from configuration than from the previous read community-list?

Reply:
The CommunitySwitch component reads the community list and is the only one that can access it unless I pass it on. But I see no reason for that. Outside the select component I only need the choosen community.